### ISSUE TITLE
Unable to install pipeline-as-a-code components via TektonOperator

### ISSUE THREAD

**Issue by AymanZahran** [tektoncd/operator#1705]
# Expected Behavior
Install Tekton Pipelines as a Code kubernetes components via the TektonOpertor

# Actual Behavior
It is only integrated with Openshift pipelines

# Steps to Reproduce the Problem
1. Install the Tekton Operator
`kubectl apply -f https://storage.googleapis.com/tekton-releases/operator/latest/release.yaml`

**Comment by jkandasa** [comment]
@AymanZahran do you want to install pipelines-as-code in kubernetes platform(not OpenShift)?

**Comment by AymanZahran** [comment]
> @AymanZahran do you want to install pipelines-as-code in kubernetes platform(not OpenShift)?

Yes, is there is a way currently to do that via the tekton operator

**Comment by jkandasa** [comment]
@AymanZahran My guess, Pipelines-as-code component is restricted to Openshift environment.

@vdemeester  @chmouel can you please confirm?
if yes, Is it possible to extend the support kubernetes platform?

**Comment by vdemeester** [comment]
@AymanZahran @jkandasa I think it is restricted to OpenShift environment today yes. It could definitely be possible to extend the support to the kubernetes platfrom though yes.

**Comment by chmouel** [comment]
just to clarify, pipelines-as-code runs well under kubernetes/kind, the operator currently only installs on openshift and this needs to be extended..

**Comment by l-qing** [comment]
Is there a plan to address this matter?

**Comment by aThorp96** [comment]
Bumping this thread. Since Pipelines as Code has migrated upstream from the Openshift Pipelines org to the TektonCD org, it seems this should be a priority if it's not already supported

**Comment by aThorp96** [comment]
It looks like this may be complete by https://github.com/tektoncd/operator/pull/3337
